tests/suites/test_suite_ssl.function

```diff
@@ -77,20 +77,33 @@ int mbedtls_test_buffer_put( mbedtls_test_buffer *buf,
         return ( input_len == 0 ) ? 0 : -1;
     }
 
-    /* Calculate the number of bytes that need to be placed at lower memory
```

Comment:
I would recommend applying "Fix segmentation fault in mbedtls_test_buffer" as a fixup to "Add a test for mbedtls_test_buffer that fails and has to be fixed". It's best if each commit is self-contained and builds and is correct.

Reply:
done